src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -644,8 +646,11 @@ internal void InsertEntry(NodeSearchElementViewModel entry, IEnumerable<string>
         private void AddEntryToExistingCategory(NodeCategoryViewModel category,
             NodeSearchElementViewModel entry)
         {
-            category.RequestBitmapSource += SearchViewModelRequestBitmapSource;
-            category.Entries.Add(entry);
+            category.RequestBitmapSource += SearchViewModelRequestBitmapSource; 
+            //Check if the category exists already. This check is vital if 
+            // the category name is updated. ex : clockwork package
```

Comment:
can you make this example a little more detailed, how was the category transformed? x => y? The package author made this change? I think that clarification would be useful to readers of this code.